python/paddle/distributed/launch.py

```diff
@@ -11,100 +11,54 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
+paddle.distributed.launch is a module that spawns multiple distributed 
+process on each trainning node for gpu trainning.
+
+Usage:
+    In both of single node training or multiple node training, this module 
+launch a process on each of the given gpu card.
+    
+    1. for single node four cards training
+       python -m torch.distributed.launch --selected_gpus="0,1,2,3,4" \
+         your_training_py (arg1 arg2 and all others)
+
+    2. for mulitple node training such as two node:192.168.0.16, 192.168.0.17
+        on 192.168.0.16:
+            python -m torch.distributed.launch --node_ips="192.168.0.16, 192.168.0.17" \
```

Comment:
is it necessary to use a comma AND a blank as the delimiter between ips IN THIS EXAMPLE? Users may get confused about whether to follow this format strictly util they read the code

Reply:
It's a common choice to use a comma.

Reply:
yeah, i get the comma is necessary, but the blank is not.